packages/patternlab/styleguide/source/assets/js/modules/feedbackFormValidation.js

```diff
@@ -13,21 +13,21 @@
   },
   {
     name: "email",
-    re: /\b[\w\.-]+@[\w\.-]+\.\w{2,4}\b/i
+    re: /\b[\w\.-]+@[\w\.-]+\.[a-z]{2,}\b/i
```

Comment:
@arthurbaghdas since US residents are not always English speakers, it is very likely that we can have non-Latin users on the website.
So we need to include the validation for both Latin and other languages for TLD. 
Of course, need to confirm that with @joeg8612 .
Some good infographics here:
https://www.icann.org/sites/default/files/assets/idn-access-domain-names-03sep15-en.pdf